Examples/Examples/OverviewMapExampleView.swift

```diff
@@ -131,9 +133,6 @@ private extension Symbol {
 }
 
 private extension Map {
-    /// A custom map for the `OverviewMap` used in a MapView.
-    static let customOverviewMapForMapView = Map(basemapStyle: .arcGISDarkGray)
-
-    /// A custom map for the `OverviewMap` used in a SceneView.
-    static let customOverviewMapForSceneView = Map(basemapStyle: .arcGISDarkGray)
+    /// A custom map for the `OverviewMap`.
+    static let customOverviewMap = Map(basemapStyle: .arcGISDarkGray)
```

Comment:
This really isn't great because if it is ever used, it will live on in perpetuity. That may be fine for an examples app, but probably worth adding a comment that in a production app, the map should be stored on a state object and not in a static property.

Reply:
I moved this to a property in the two places that need it,